cw_infrastructure/cw_infrastructure/validators.py

```diff
@@ -296,6 +309,18 @@ def resolvable_ip_or_domain_name_with_port_validator(name, value):
     return resolvable_ip_or_domain_name_validator(name, stem)
 
 
+def resolvable_ip_or_domain_name_opt_port_validator(name, value):
```

Comment:
Hmm, this makes calls to this quite long. I wonder if we should just call it `ip_or_domain_name_opt_port_validator`, and dispense with distinguishing in the function name with it's resolvable. That would also match `ip_or_domain_name_opt_port_list_validator` above.